imports/config/EMailTemplates.js

```diff
@@ -0,0 +1,32 @@
+import { Accounts } from 'meteor/accounts-base'
+
+Accounts.emailTemplates.siteName = "4Minitz";
+Accounts.emailTemplates.from     = "4Minitz <mail@4minitz.com>";
```

Comment:
This should not be hard coded. Instead, there should be a setting for this. Maybe re-use defaultEmailSenderAddress? Same applies to the siteName and other uses of this mail address below.